pinata/doc.md

```diff
@@ -0,0 +1,116 @@
+# How to use Bifrost with Pinata Cloud
+Welcome to the Bifrost documentation for Pinata Cloud! In this guide, we'll show you how to use Bifrost to upload files to Pinata Cloud.
+
+## Overview
+Pinata Cloud is a powerful decentralized cloud storage platform that enables users to securely store and manage digital assets. It allows you to store and access your data on IPFS. Bifrost provides a simple and intuitive way to upload files to Pinata Cloud without having to write complex code.
+
+## Prerequisites
+Before you can start using Bifrost to upload files to Pinata Cloud, you'll need to make sure you have the following:
+- A Pinata Cloud account with API access
+- A Pinata Cloud API key and API secret
+- Bifrost installed on your local machine
+
+### Steps:
+#### Create a Pinata Cloud API key
+1. Sign up for a Pinata Cloud account if you haven't already
+2. Go to the API Keys page and create a new API key
+3. Download the API key's JSON file and save it to a secure location
+_Note the name of your bucket as you will need it later_
```

Comment:
Like it is with google cloud above, we can remove lines 13 to 18 as we are trying to be concise in order to solely sell our solution.